python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -271,9 +271,12 @@ class RandomForestRegressor(Base):
     quantile_per_tree : boolean (default = False)
                         Whether quantile is computed for individal trees in RF.
                         Only relevant for GLOBAL_QUANTILE split_algo.
+    seed : int (default = -1)
+           If int, then the seed is used by the models random number
```

Comment:
`Seed for the random number generator. Unseeded by default. Does not currently fully guarantee the exact same results.` 

Reads much better, and we should inform users about the limitation